google-cloud-tools-plugin/src/com/google/cloud/tools/intellij/appengine/cloud/executor/AppEngineStandardDeployTask.java

```diff
@@ -109,24 +109,21 @@ public void execute(ProcessStartListener startListener) {
   ProcessExitListener deploy(
       @NotNull final Path stagingDirectory,
       @NotNull final ProcessStartListener startListener) {
-    return new ProcessExitListener() {
-      @Override
-      public void onExit(int exitCode) {
-        if (exitCode == 0) {
-          try {
-            deploy.deploy(stagingDirectory, startListener);
-          } catch (RuntimeException re) {
-            deploy.getCallback()
-                .errorOccurred(GctBundle.message("appengine.deployment.exception") + "\n"
-                    + GctBundle.message("appengine.action.error.update.message"));
-            logger.error(re);
-          }
-        } else {
+    return (exitCode) -> {
+      if (exitCode == 0) {
+        try {
+          deploy.deploy(stagingDirectory, startListener);
+        } catch (RuntimeException re) {
           deploy.getCallback()
-              .errorOccurred(GctBundle.message("appengine.deployment.error.during.staging", exitCode));
-          logger.warn(
-              "App engine standard staging process exited with an error. Exit Code:" + exitCode);
+              .errorOccurred(GctBundle.message("appengine.deployment.exception") + "\n"
```

Comment:
Yeah, I think so. Let's hope I don't have to rebase a lot more so these don't come up more often :)